### PR TITLE
Raji. New new button

### PR DESCRIPTION
**Pull request by Tobi2Unique**
New button

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -37,4 +37,9 @@ button {
   width: 50%;
   cursor: pointer;
   margin: auto;
+}
+
+.gitButton{
+	 background-color: green;
+  color: white;
```
Comment (ngozi-ekekwe) [inline index.css:44]: Please Fix the indentation on this line.

File: index.html [hunk 2]
```diff
@@ -24,6 +24,7 @@ <h2 class="text-align-center">Github Example</h2>
             <button>Submit</button>
             <button>Submit</button>
             <button>Submit</button>
+			<button class="gitButton">Submit Raji</button>
```
Comment (ngozi-ekekwe) [inline index.html]: I like this color

**Review by ngozi-ekekwe — CHANGES_REQUESTED** [review]

**Review by ngozi-ekekwe — COMMENTED** [review]
Fix all the comments I left then I can merge your PR

File: index.css (hunk 1 above)
Comment (Tobi2Unique) [reply index.css:44]: Thanks

File: index.html (hunk 2 above)
Comment (Tobi2Unique) [reply index.html]: Nagode

**Pull request merged**